graylog2-web-interface/package.json

```diff
@@ -178,6 +177,7 @@
   "resolutions": {
     "@types/react": "18.3.12",
     "nth-check": "2.0.1",
-    "typescript": "5.8.2"
+    "typescript": "5.8.2",
+    "jquery": "3.5.1"
```

Comment:
We need to pin this for now, because removing `toastr` results in an update of `jquery` to `3.7.x`, which will break `typeahead.js`. This will be fixed with the removal of `typeahead.js`.